### DISCUSSION TITLE
CPF	Opłata za monety - przypisanie order.id w /billing/billing-entries

### DISCUSSION THREAD

**Discussion by tomylee0** [allegro/allegro-api#10273] [Suggestions - resolved]
Billing "Opłata za monety" jest pobierany dla konkretnego zamówienia. A nie ma przypisanego order.id. Powinien mieć abyśmy mogli lepiej analizować konkretne zamówienia.

**Comment by PrzemyslawLukanowski** [comment]
I tutaj również dziękuję za sugestię i również wrócimy z informacją, jeśli uda się wdrożyć taką informację.

**Comment by PrzemyslawLukanowski** [comment]
Zaktualizowałem status sugestii jako "wdrożona" - aktualnie zwracamy już order.id dla wpisów CPF.
